src/cpu.c

```diff
@@ -133,6 +133,12 @@ int clockCpu(struct CPU *cpu, struct BUS *bus) {
     signed int compareResult = 0;
 
     switch (opcode->instruction->index) {
+      case BIT:
+        compareResult = readBus(address, bus);
+        cpu->status.negative = (compareResult >> 7) & 1;
```

Comment:
Is the `& 1` needed at this point?
7 being the last bit I think it's already what we are looking for and no need to "reduce" it with a bitwise.
Look:
![image](https://user-images.githubusercontent.com/6990857/112137647-45b59d80-8baf-11eb-893e-5bf3a9c483f9.png)

For the value 0xaa = 10101010

10101010 >> 7 = 1 (dont need to evaluate the & I think)

But 10101010 >> 6 = 10 (we must evaluate the & 1) 
10 & 1 = 0

Reply:
:+1: